deploy.go

```diff
@@ -372,8 +373,15 @@ deployment. Provide a relative path to your file.`,
 			log.Fatal(err.Error())
 		}
 
-		// Destination path
-		remotePath := path.Join(common.DaemonProjectPath, args[0])
+		// Get flag for destination
+		dest, err := cmd.Flags().GetString("dest")
+		if err != nil {
+			log.Fatal(err.Error())
+		}
+
+		// Destination path - todo: allow config
+		projectPath := "/app/host/inertia/project"
```

Comment:
Quick question: Why does the path start with `/app/host`?

Reply:
in `client/scripts/daemon-up.sh`, we mount the host home directory to there:

```bash
sudo docker run -d \
    --restart unless-stopped \
    -p "$DAEMON_PORT":"$CONTAINER_PORT" \
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v "$HOME":/app/host \ # <--- relevant line
    -e HOME="$HOME" \
    -e SSH_KNOWN_HOSTS='/app/host/.ssh/known_hosts' \
    --name "$DAEMON_NAME" \
    "$IMAGE" "$HOST_ADDRESS" > /dev/null 2>&1
```

so on the host, `$HOME/inertia/project` would map to `/app/host/inertia/project`. Interestingly enough, we have to do some finnicking around that when we execute Docker commands - because the socket is on the host, we need to translate `/app/host/` paths back to `$HOME/` paths 😅 See https://github.com/ubclaunchpad/inertia/blob/v0.4.0-rc2/daemon/inertiad/project/build.go#L36

Reply:
![image](https://user-images.githubusercontent.com/11564004/41804937-73773148-7654-11e8-9be2-aed110774381.png)